editor/src/components/inspector/code-element-section.spec.browser2.tsx

```diff
@@ -117,18 +117,12 @@ describe('The Inspector code element section', () => {
     expect(inspectorSectionsContainer.children.length).toEqual(1)
   })
 
-  it('Is the only section displayed when a map is selected', async () => {
-    const editor = await renderTestEditorWithCode(testProjectCode, 'await-first-dom-report')
-    await clickElementWithTestId(editor, 'ddd')
-
-    // Check that the code element section is displayed
-    const codeElementSection = editor.renderedDOM.getByTestId(CodeElementSectionTestId)
-    expect(codeElementSection).toBeDefined()
+  // it('Is not displayed when a map is selected', async () => {
```

Comment:
Does this test still need to be commented out?

Reply:
Probably shouldn't be.

Reply:
Restored in 83ae48b86.